streaming/src/test/run_streaming_queue_test.sh

```diff
@@ -44,15 +44,22 @@ if [ ! -d "$RAY_ROOT/python" ]; then
   exit 1
 fi
 
-REDIS_MODULE="./bazel-bin/libray_redis_module.so"
-REDIS_SERVER_EXEC="./bazel-bin/external/com_github_antirez_redis/redis-server"
-STORE_EXEC="./bazel-bin/plasma_store_server"
-REDIS_CLIENT_EXEC="./bazel-bin/redis-cli"
-RAYLET_EXEC="./bazel-bin/raylet"
-STREAMING_TEST_WORKER_EXEC="./bazel-bin/streaming/streaming_test_worker"
-GCS_SERVER_EXEC="./bazel-bin/gcs_server"
-
-# Allow cleanup commands to fail.
+REDIS_MODULE="$RAY_ROOT/bazel-bin/libray_redis_module.so"
```

Comment:
Remove this script. Combine all streaming queue tests with travis yaml.

Reply:
I'll do this in another PR.